conditionals.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var year = prompt("¿how old are you?");
```

Comment:
In english, question marks are not to being used to start a question. Also use capital letter:
```suggestion
    var year = prompt("How old are you?");
```